Editor/Window/GameLiftPlugin.cs

```diff
@@ -84,8 +91,9 @@ private VisualElement CreateContentContainer(Pages page, VisualElement contentCo
             return container;
         }
 
-      private void OpenTab(string tabName)
+        private void OpenTab(string tabName)
         {
+            StateManager.LastOpenTab = button.name;
```

Comment:
I moved this down to this helper method because if any other tab calls the `internal void OpenTab` method, it wouldn't update the state before